Content.Server/Administration/Commands/BanMassCommand.cs

```diff
@@ -0,0 +1,117 @@
+using System.Linq;
+using Content.Server.Administration.Managers;
+using Content.Shared.Administration;
+using Content.Shared.CCVar;
+using Content.Shared.Database;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Content.Server.Database;
+using Robust.Shared.Console;
+
+namespace Content.Server.Administration.Commands;
+
+[AdminCommand(AdminFlags.MassBan)]
+public sealed class BanMassCommand : LocalizedCommands
+{
+
+    [Dependency] private readonly IPlayerLocator _locator = default!;
+    [Dependency] private readonly IBanManager _bans = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    [Dependency] private readonly IPlayerManager _playerManager = default!;
+    [Dependency] private readonly ILogManager _logManager = default!;
+    [Dependency] private readonly IServerDbManager _dbManager = default!;
+
+    public override string Command => "banmass";
+
+    public override async void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        string reason;
+        uint minutes;
+
```

Comment:
```suggestion
```
Don't declare vars ahead of time, just do `var reason = ..` and `out var minutes`.

Reply:
yes